doc/misc/npm-faq.md

```diff
@@ -7,18 +7,19 @@ npm-faq(7) -- Frequently Asked Questions
 
     npm config set viewer browser
 
-to open these documents in your default web browser rather than `man`.
+Open the following documents in your default web browser rather than `man`.
```

Comment:
This doesn't make sense to me. It seems to refer to documents outside of this one, not the "following", i.e. anything below.